site/en/docs/privacy-sandbox/faq/testing-faq/index.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: 'layouts/doc-post.njk'
+title: 'Privacy Sandbox testing FAQs'
+subhead: >
+   Answers to frequently asked questions related to Privacy Sandbox testing.
+description: >
+   Answers to frequently asked questions related to Privacy Sandbox testing.
+date: 2023-10-02
+---
+
+**What are the key coordination areas between DSPs and SSPs for the Protected Audience API?**
+
+Below are the key aspects that require alignment:
+
+-   **Creative audit**: Does the SSP have a creative pre-registration endpoint for scanning, and will DSP support that workflow? If not, discuss with the SSP what they require from the DSP to implement creative audit for publisher controls.
+-   **DSP's `renderURL` methodology**. Some SSPs may require the `renderURL` to include metadata such as `seatID` to support scoring and billing. How the DSP structures `renderURL` has implications for critical SSP use cases.
+-   **ORTB**
+    -   ORTB 2.X bid request: How the SSP will signal PA-eligible auction to the DSP.
+    -   ORTB 2.X bid response: How the DSP will respond to the SSP for PA auction.
+-   **PA on-device bidding data**: SSP to DSP
+    -   `auctionSignals`: Data the SSP includes in the `auctionConfig` that is made available to all DSP bidding functions.
+-   **PA on-device scoring data**: DSP to SSP
+    -   Bid: How to handle multi-currencies. Each SSP may have different requirements.
+    -   Ad object (returned from `generateBid()`): The SSP may use this for scoring and publisher controls. DSPs and SSPs need to align on what data is included and on a data structure.
+-   **PA win reporting**: SSP to DSP
+    -   What SSP's `reportResult()` will make available to the DSP's `reportWin()`.
+-   **Post-auction reporting**: DSP to SSP. Context: The DSP must capture and delegate post-auction reports to the SSP. If the SSP needs clicks, views, viewability metrics, the DSP must enable them to receive these events.
+    -   What reports the SSP will need.
+    -   The methodology for measuring that event (such as, viewability definition).
+    -   DSP implementation of the `reportEvent()` call to support the SSP requirements.
+    -   SSP implementation of the `registerAdBeacon()` lining up the event naming with what the DSP will trigger in the creative.
+
+**Are the Attribution Reporting API and the Aggregation Service ready and available for testing?**
+
+The Attribution Reporting API will be generally available and ramp-up is already in progress. With that, Attribution Reporting will soon be available for use on 100% of traffic. Note that Attribution Reporting can be used with all ads, not just those served by Protected Audience.
```

Comment:
I feel like if a question asks about the Aggregation Service, the response should at least mention it once, even if the assumption is people understand the context.

Reply:
Good catch.

Reply:
Asking now

Reply:
still waiting. Might get answer today.